src/Row.ts

```diff
@@ -0,0 +1,45 @@
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+import { RegistryMixin, RegistryMixinProperties } from '@dojo/widget-core/mixins/Registry';
+import { v, w } from '@dojo/widget-core/d';
+import { HasColumns } from './interfaces';
+import { CellProperties } from './Cell';
+import { theme, ThemeableMixin } from '@dojo/widget-core/mixins/Themeable';
+
+import * as rowClasses from './styles/row.css';
+
+export interface RowProperties extends WidgetProperties, HasColumns, RegistryMixinProperties {
+	item: any;
+}
+
+@theme(rowClasses)
+class Row  extends ThemeableMixin(RegistryMixin(WidgetBase))<RowProperties> {
```

Comment:
There's lots of info in the dojo 2 room and I also raised an issue on typescript, I don't have the link to hand (I'll add later).

Basically you cannot extend an expression, which Mixin(Class)<Properties> actually is. With the release of 2.2.1 this would actually fail to create the d.ts file when running `grunt dist`.

There's an extra space after `Row` and before `extends`... just killing my OCD.